avm/res/authorization/role-assignment/mg-scope/version.json

```
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "0.2",
```

Must be merged as 0.1
```suggestion
  "version": "0.1",
```